src/components/control-panel.tsx

```diff
@@ -79,7 +70,7 @@ export const ControlPanel: React.SFC<IControlPanelProps> = (props) => {
               disabled={disableDuration}>
         {[durationOptions]}
       </Select>
-      {props.isConnectorAwake ? startCollectingButton : startConnectingButton}
```

Comment:
There's an error message that refers to the "Launch SensorConnector" button at the bottom of the page that should be changed as well. The problem with removing this button is that if you cancel in that dialog there's no easy way to get to a place where you can launch the SensorConnector other than hitting the reset/reload button.

Reply:
I see your point.  Dan voted to remove it so I think we're overruled.  But, at least for now, I'll update the message.

Reply:
I updated my original comment around the time you were replying to point out that the Connect to Wired Sensor button may now provide the necessary functionality.